src/Events/Controllers/PushToInboundController.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Threading.Tasks;
+
+using Altinn.Common.AccessToken.Configuration;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Services.Interfaces;
+using AutoMapper;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Provides operations for handling events
+    /// </summary>
+    [Authorize]
+    [Route("events/api/v1/push/inbound")]
```

Comment:
I'm not sure it's safe to rename the endpoint like this. What's the plan?

Reply:
inboud should be a part of the route directly on the method. 
So  [Route("events/api/v1/push")] on the controller

And on the method     [Route("inbound")]

Reply:
I'm worried about the removal of `/events/api/v1/push`. It's being replaced with `events/api/v1/push/inbound`, but I want a plan for that change.

Reply:
ohh!! Okay, the new endpoint is supposed to be in addition to the one we've removed.